websites/T/Tempest Fansub/presence.ts

```diff
@@ -0,0 +1,136 @@
+const presence = new Presence({
+		clientId: "1047102386478534727",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const { pathname } = window.location,
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/MVWiX8p.png",
+		},
+		_host = location.host;
+	//Dino
+	if (pathname === "/dinozor-oyunu/") presenceData.details = "Dino oynuyor.";
+	//Anime arama
+	else if (
+		location.search.includes("?s=") &&
+		location.host === "tempestfansub.com"
+	) {
+		presenceData.details = "Bir anime arıyor.";
+		presenceData.state = location.search.replace("?s=", "");
+		//Animeler
+	} else if (
+		(_host === "tempestfansub.com" && pathname === "/") ||
+		pathname.endsWith("/anime/")
+	)
+		presenceData.details = "Animelere göz atıyor.";
+	//Bir Anime
+	else if (!pathname.endsWith("/anime/") && pathname.includes("/anime/")) {
+		presenceData.details = "Bir animeye bakıyor.";
+		presenceData.state = document
+			.querySelector("h1.entry-title")
+			?.textContent.trim();
+		const imgs = document.querySelectorAll(
+			"#content > div > div > article > div:nth-child(2) > div > div.thumbook > div > img"
+		);
+		for (const element of imgs) {
+			if (element.getAttribute("src") !== null)
+				presenceData.largeImageKey = element.getAttribute("src");
+		}
+		//Manga arama
+	} else if (
+		location.search.includes("?s=") &&
+		location.host === "manga.tempestfansub.com"
+	) {
+		presenceData.details = "Bir manga arıyor.";
+		presenceData.state = location.search.replace("?s=", "");
+		//Mangalar
```

Comment:
Use URLSearchParams or URL.searchParams

Reply:
its not working

Reply:
You aren't using it properly then, it will work.